js/src/modals/index.js

```diff
@@ -14,44 +14,26 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
-import AddAddress from './AddAddress';
-import AddContract from './AddContract';
-import CreateAccount from './CreateAccount';
-import CreateWallet from './CreateWallet';
-import DappPermissions from './DappPermissions';
-import DappsVisible from './AddDapps';
-import DeleteAccount from './DeleteAccount';
-import DeployContract from './DeployContract';
-import EditMeta from './EditMeta';
-import ExecuteContract from './ExecuteContract';
-import FirstRun from './FirstRun';
-import LoadContract from './LoadContract';
-import SaveContract from './SaveContract';
-import Shapeshift from './Shapeshift';
-import Verification from './Verification';
-import Transfer from './Transfer';
-import PasswordManager from './PasswordManager';
-import UpgradeParity from './UpgradeParity';
-import WalletSettings from './WalletSettings';
-
-export {
-  AddAddress,
-  AddContract,
-  CreateAccount,
-  CreateWallet,
-  DappPermissions,
-  DappsVisible,
-  DeleteAccount,
-  DeployContract,
-  EditMeta,
-  ExecuteContract,
-  FirstRun,
-  LoadContract,
-  SaveContract,
-  Shapeshift,
-  Verification,
-  Transfer,
-  PasswordManager,
-  UpgradeParity,
-  WalletSettings
-};
+export AddAddress from './AddAddress';
```

Comment:
This was a mistake, did this along with `~/ui/Icons` & `~/views` to not have merge conflicts, or rather, make them easier to resolve. So it shouldn't strictly be in here, but does do the right job and aligns the codebase with new master.